charts/nginx/templates/nginx-deployment.yaml

```diff
@@ -36,12 +36,6 @@ spec:
       containers:
         - name: nginx
           securityContext:
-            allowPrivilegeEscalation: true
-            capabilities:
-              add:
-                - NET_BIND_SERVICE
-              drop:
-                - ALL
```

Comment:
changing the default container ports of nginx to non privileged ports ought to remove this requirement, but this was not tested on a normal kube cluster with psp enabled. It will be discovered in dev, where psps are enabled, if there is an issue.

Reply:
@jpweber this part is questionable I think, i'm not sure if we ought to drop or not. Just let me know what you think. If we do drop it, and it causes an issue we can always do a quick patch - there is very little chance of this slipping under the radar because if it's broken I think ingress won't work.

Reply:
I think this is a great idea - the less privilege we need the better!